cloudinit/userdata_cloudinit.erb

```diff
@@ -41,8 +41,19 @@ users:
    lock-passwd: false
    passwd: <%= @host.root_pass %>
 
+<% unless @host.param_false?('package_upgrade') -%>
+package_upgrade: true
+<% end -%>
+
 <%# Contact Foreman to confirm instance is built -%>
 phone_home:
- url: <%= foreman_url('built') %>
- post: []
- tries: 10
+  url: <%= foreman_url('built') %>
+  post: []
+  tries: 10
+
+<% unless @host.param_false?('reboot') -%>
```

Comment:
OK, I agree with that.  Seems like more will not want this in a cloud environment.